client/state/login/magic-login/actions.js

```diff
@@ -111,3 +111,7 @@ export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
 			} );
 		} );
 };
+
+export function goBackToWordPressDotCom() {
```

Comment:
> I'm not sure if state is the best place to put this in, since this doesn't directly work with the Redux state.

The decision to put it in the actions file was based on @mcsf's suggestion:

> don’t create the function in `render`, maybe make it a pseudo-action so that you can reuse it in both components

Happy to move it if you can think of a better place, but technically, route changes are actions affecting app state, if you want to get technical :P